requirements.txt

```diff
@@ -1,6 +1,7 @@
 click==6.2
 PTable==0.9.2
 netaddr==0.7.18
+nsot==1.1.1
```

Comment:
I don't think this is the appropriate place for this, because it means that `nsot` would always be pulled in as a part of any `pynsot` install!